jdi-light/src/main/java/com/epam/jdi/light/actions/ActionHelper.java

```diff
@@ -83,18 +83,19 @@ public static String fillTemplate(String template,
             } else if (template.contains("%s")) {
                 template = format(template, getArgs(jp));
             }
+            String newTemplate = null;
```

Comment:
It wont fix the issue, because you have another two reassignments just a few lines above. 
In this case it will be better to declare temporal variable at the start:
`String newTemplate = template;`
and use this new variable everywhere. 
It also helps to avoid redundant complexity, such as:
`return newTemplate != null ? newTemplate : template;`